ui/app/pages/swaps/build-quote/build-quote.js

```diff
@@ -93,11 +98,21 @@ export default function BuildQuote({
   );
   const memoizedUsersTokens = useEqualityCheck(usersTokens);
 
+  const queryFromToken =
```

Comment:
These changes seem unrelated to notifications :thinking:

Reply:
This is to maintain the behaviour of having ETH selected by default after going to the swaps route from the intro popup. This is why this code is in the swaps related action function in `WhatsNewPopup`:
```
 global.platform.openExtensionInBrowser(
        BUILD_QUOTE_ROUTE,
        `fromAddress=${ETH_SWAPS_TOKEN_OBJECT.address}`,
      );
```

Reply:
Ah, I see. So the functional change is that it now opens in a separate tab - is that correct? And that's something that we want?

Reply:
Given that we are no longer having a notification on swaps in the extension ([see this thread](https://consensys.slack.com/archives/G01PKA4QC3C/p1619206421011800)), there is no longer a need for this. It has been removed as part of 9dbdb7661, which removes all code required for the notification that would bring the user to the swaps route.